From 473608c21ffa824026f93c8f67c92c302f2cd2bb Mon Sep 17 00:00:00 2001
From: Christophe Kerello <christophe.kerello@foss.st.com>
Date: Fri, 20 Dec 2024 14:33:05 +0100
Subject: [PATCH 769/954] fix(stm32mp2-fdts): set SDMMC max frequency to 166
 MHz on stm32mp21

Set SDMMC max frequency to 166 MHz on stm32mp21.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git c643b9576277ae13941048b55a878c504bec7bc6 fix(stm32mp2-fdts): set SDMMC max frequency to 166 MHz on stm32mp21]
Change-Id: I5f568e10a3ab48ebd6ec2c3c0fb170d98638b49c
Signed-off-by: Christophe Kerello <christophe.kerello@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/426154
Domain-Review: Christophe KERELLO <christophe.kerello@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Christophe KERELLO <christophe.kerello@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Christophe KERELLO <christophe.kerello@st.com>
---
 fdts/stm32mp211.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/fdts/stm32mp211.dtsi b/fdts/stm32mp211.dtsi
index b951dc5398..4cf300fe47 100644
--- a/fdts/stm32mp211.dtsi
+++ b/fdts/stm32mp211.dtsi
@@ -254,7 +254,7 @@
 				resets = <&rcc SDMMC1_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				status = "disabled";
 			};
 
@@ -267,7 +267,7 @@
 				resets = <&rcc SDMMC2_R>;
 				cap-sd-highspeed;
 				cap-mmc-highspeed;
-				max-frequency = <120000000>;
+				max-frequency = <166000000>;
 				status = "disabled";
 			};
 		};
-- 
2.53.0

